ui_tests/src/test/resources/com/saucedemo/brokenTests.feature

```diff
@@ -50,7 +50,7 @@ Feature: Mixed Broken Tests
     And I click 'Sign In' button on Sign in page
     And I click 'ADD TO CART' button on 'Sauce Labs Bolt T-Shirt' product on Products page
     And I click on cart icon on Header
-    And I click 'Checkout' button on Cart page
+    And I click on menu icon on Header
     When I click 'Logout' link on Menu
     Then I am on Sign in page
```

Comment:
@masrur18 Please make sure that this scenario is doing what it supposed to be doing as per the:
`Scenario: Sign out from cart your information page`

Reply:
Done...Thank you for help.